sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ApproximateUniqueTest.java

```diff
@@ -276,6 +301,50 @@ public void testApproximateUniqueWithDifferentSampleSizes() {
     }
   }
 
+  /**
+   * Test ApproximateUniqueCombineFn. TestPipeline does not use combiners.
```

Comment:
It does (otherwise how would it evaluate the Combine transform), just not as extensively as CombineFnTester.

Reply:
I should clarify, it does not seem to merge() at all. I had changed merge() to return null, and it had no effect. In that sense it does not combine. I will update the comment.

Reply:
Updated the comment.